assets/src/edit-story/components/library/panes/shared/pillGroup/pillGroup.js

```diff
@@ -223,11 +222,11 @@ const Media3pCategories = ({
   );
 };
 
-Media3pCategories.propTypes = {
+PillGroup.propTypes = {
   categories: PropTypes.array.isRequired,
   selectedCategoryId: PropTypes.string,
   selectCategory: PropTypes.func,
   deselectCategory: PropTypes.func,
```

Comment:
Perhaps `category` is not the best generic term here? Maybe just `item` or something similarly vague?